src/test/java/TaskTest.java

```diff
@@ -0,0 +1,25 @@
+import duke.Todo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TaskTest {
+    Todo t = new Todo("123");
+
+    @Test
+    void testGetStatusIcon() {
```

Comment:
you can use underscores to name the test methods: eg test_Find_Success.

Reply:
Ok, I will adjust the names